README.md

```diff
@@ -36,96 +20,95 @@ After you're finished please remove all the comments and instructions!
   - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
 - [👥 Authors](#authors)
 - [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [oop-school-library] <a name="about-project"></a>
+# 📖 Decode Morse <a name="about-project"></a>
 
-**[oop-school-library]** Is a Library manager for a school.
+> Decode Morse is a project in Ruby that decodes morse messages.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
+> This project uses the following stack :
+
 <details>
-<summary>Back-end</summary>
-  <ul>
-    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
-  </ul>
+  <summary>Ruby</summary>
 </details>
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-- **[manages which persons takes books from the library and which books are lend by the library identifying if it's a student or a teacher]**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-> Add a link to your deployed project.
+<!-- > Describe between 1-3 key features of the application. -->
 
-- [Live Demo Not Available]()
+- **Decode Morse messages**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
+<!-- > Clone the repository by clicking on the 'Code' button and copy the link -->
+
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
 In order to run this project you need:
 
-ruby installed 
+You should have Ruby install on your local machine
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-  git clone https://github.com/Swarzstein/oop-school-library.git
-  cd oop-school-library
+Example commands:
 
+```sh
+  cd my-folder
+  git clone https://github.com/iqbal-elham/code-morse-decoder.git
+```
 
 ### Install
 
-Install this project with:
+To install ruby go to : [Install Ruby](https://rubyinstaller.org/) and select your operating system.
 
-  bundle install
 
 ### Usage
 
-To run the project, execute the following command:
+To run the project, open your project in your terminal  and type ruby morse_decoder.rb .
+
+Example command:
 
-ruby app.rb
+```sh
+  ruby morse_decoder.rb
+```
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- AUTHORS -->
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
-
 👤 **Juan Zorrilla**
 
 - GitHub: [@Swarzstein](https://github.com/Swarzstein)
 - LinkedIn: [Juan Zorrilla](https://www.linkedin.com/in/juan-a-zorrilla/)
 
+👤 **Iqbal Elham**
+
+- GitHub: [@iqbal-elham](https://github.com/Iqbal-Elham)
+- linkedIn: [@iqbal-elham](https://www.linkedin.com/in/iqbal-elham)
 
 
```

Comment:
![image](https://github.com/Swarzstein/morse-code/assets/93039359/e16c1d68-64f9-4c5f-a4b3-3ccebb74d0c6)

- [ ] Kindly remove this section description to make your readme file descriptive and clean 👍